src/pages/Chat/Chat.styled.jsx

```diff
@@ -116,7 +116,7 @@ export const FooterChatInput = styled.form`
 export const MessageRow = styled.div`
   margin: 10px 0;
   display: flex;
-  flex-direction: row-reverse;
+  flex-direction: ${ props => props.client ? 'row-reverse' : 'row'};
```

Comment:
Good work! Please check code style

```suggestion
  flex-direction: ${props => props.client ? 'row-reverse' : 'row'};
```

or

```suggestion
  flex-direction: ${ props => props.client ? 'row-reverse' : 'row' };
```